### PR TITLE
Remove duplicated ScheduledTask

### PR DESCRIPTION
**Pull request by mario-galic**
@tomverran @rtyley 

Use ScheduledTask from membership-common.

### CODE REVIEW

**Comment by afiore** [comment]
As the akka actorSystem is global, perhaps we could define its implicit as part of  the main [config object](https://github.com/guardian/membership-frontend/blob/master/frontend/app/configuration/Config.scala).

**Comment by afiore** [comment]
:+1: overall

File: frontend/app/configuration/Config.scala [hunk 1]
```diff
@@ -183,4 +184,8 @@ object Config {
 
   def digipackRatePlanIds(env: String) = DigitalPackRatePlanIds.fromConfig(
     config.getConfig(s"touchpoint.backend.environments.$env.zuora.ratePlanIds.digitalpack"))
+
+  object Implicits {
+    implicit val akkaSystem = Akka.system
```
Comment (rtyley) [inline frontend/app/configuration/Config.scala]: Surprised this is necessary, but ok!

**Comment by rtyley** [comment]
LGTM

**Pull request merged**